integrations/clickup/client.py

```diff
@@ -0,0 +1,190 @@
+from typing import Any, AsyncGenerator, Dict, List
+
+from httpx import Timeout
+
+from loguru import logger
+
+
+from port_ocean.utils import http_async_client
+
+
+class ClickUpClient:
+    def __init__(self, clickup_token: str) -> None:
+        self.clickup_url = "https://api.clickup.com"
+        self.clickup_token = clickup_token
+        self.clickup_api_url = f"{self.clickup_url}/api/v2"
+
+        self.headers = {"Authorization": self.clickup_token}
```

Comment:
You can use the Auth header directly in the self.client.headers.update. You don't need to have it as a class variable